zap/src/main/java/org/zaproxy/zap/extension/autoupdate/InstalledAddOnsTableModel.java

```diff
@@ -37,13 +37,13 @@ public class InstalledAddOnsTableModel extends AddOnsTableModel {
     private static final long serialVersionUID = 1L;
 
     private static final String[] COLUMN_NAMES = {
-        "", // Column for warning of running issues (e.g. incorrect Java version, missing
-        // dependency...)
+        Constant.messages.getString("cfu.table.header.note"),
+        // Column for warning of running issues (e.g. incorrect Java version, missing dependency...)
```

Comment:
This is in the wrong place now, but since it has a name drop it.

Reply:
Right, now that you mention it, that makes given that the other titles do not have comments.  I have removed the comments now